schema/partner.js

```diff
@@ -98,6 +102,41 @@ const PartnerType = new GraphQLObjectType({
           ].join(' ');
         },
       },
+      fair_highlights: {
+        type: GraphQLString,
+        args: {
+          length: {
+            type: GraphQLInt,
+            defaultValue: 3,
+          },
+        },
+        resolve: ({ _id }, { length }) => {
+          return gravity('shows', {
+            partner_id: _id,
+            displayable: true,
+            at_a_fair: true,
+            size: 30,
+          }).then(shows => {
+            let fairs = shows.map(({ fair: { name, banner_size } }) => ({
+              banner_size,
+              name: name.replace(/\d{2,4}.*$/, '').trim(),
+            }));
+
+            fairs = uniqBy(fairs, 'name');
+            fairs = sortBy(fairs, 'name');
+            fairs = sortBy(fairs, ({ banner_size }) =>
+              ['x-large', 'large', 'medium', 'small', 'x-small'].indexOf(banner_size)
+            );
```

Comment:
FYI: lodash's sorts are stable, meaning we sort by name, then by size so that same sizes are alphabetical within